Improve Documentation (#339)
hivemq/mqtt-cli: docs/_docs/subscribe.md

| `mqtt sub -t topic`                     | Subscribe to a topic with default settings and block the console.                        |
| `mqtt sub -t test1 -t test2`            | Subscribe to the topics 'test1' and 'test2' with default settings and block the console. |
| `mqtt sub -t test -h localhost -p 1884` | Subscribe to topic 'test' at a broker with the address 'localhost:1884'.                 |

***

## Options

### Subscribe Options

{% include options/subscribe-options.md %}

### Connect Options

{% include options/connect-options.md %}

#### Will Options

{% include options/will-options.md %}

#### Connect Restrictions

{% include options/connect-restrictions-options.md %}

### Security Options

#### Credentials Authentication

{% include options/authentication-options.md %}

#### TLS Authentication

{% include options/tls-options.md %}

### Logging Options

{% include options/logging-options.md %}

### Debug Options

{% include options/debug-options.md %}

### Help Options

{% include options/help-options.md defaultHelp=false %}

*** 

## Further Examples

> Subscribe to one topic with default QoS `Exactly Once`.

Reviewer: ```suggestion
Subscribe to one topic with QoS level `Exactly Once`.
```

**NOTE**: If you specify one QoS and multiple topics, the QoS will be used for all topics.

```
mqtt sub -t topic1 -t topic2 -q 2  
```

***

Subscribe to the given topics with a specific QoS for each topic.
(`topic1` will have QoS 0, `topic2` QoS 1 and `topic3` QoS 2)

```
mqtt sub -t topic1 -q 0 -t topic2 -q 1 -t topic3 -q 2
```

***

Subscribe to a topic and output the received publish messages to the file `publishes.log` in the current directory.

> **NOTE**: If the file is not created yet, it will be created by the CLI. If it is present the received publish

Reviewer: ```suggestion
**NOTE**: CLI creates the file if it does not exist. The received publish messages will be appended.
```

messages will be appended to the file.

```
mqtt sub -t topic -of publishes.log
```

***

Subscribe to a topic and output the received publish messages to the file `publishes.log` in a
specified `/usr/local/var` directory.

> **NOTE**: If the file is not created yet, it will be created by the CLI. If it is present the received publish messages

Reviewer: ```suggestion
**NOTE**: CLI creates the file if it does not exist. The received publish messages will be appended.
```
